cluster/config-defaults.yaml

```diff
@@ -677,9 +677,9 @@ teapot_admission_controller_configmap_deletion_protection_factories_enabled: "tr
 # enable the rolebinding admission-controller webhook which validates rolebindings and clusterrolebindings
 teapot_admission_controller_enable_rolebinding_webhook: "true"
 
-# enable the generic admission-controller webhook which catches all resources
-teapot_admission_controller_enable_generic_webhook: "true"
-# prevent write operations for non-admin users in protected namespaces
+# enable the generic deny-all admission webhook which rejects all requests it receives
+teapot_admission_controller_enable_write_protection_webhook: "true"
+# configure the behaviour of the deny-all admission webhook, `true` blocks everything, `false` allows everything
 teapot_admission_controller_prevent_write_operations: "true"
```

Comment:
It was mostly to turn it off quickly in case unwanted things are blocked. But I guess the other config item (that removes the webhook entirely) is equally quick (or changing the conditions). I would probably just leave it for now. It's not hurting to have it for now.

Reply:
Sure. As long as we have considered it 🙂